test/integration/controllers/etcd/reconciler_test.go

```diff
@@ -1671,3 +1677,65 @@ var _ = Describe("buildPredicate", func() {
 		)
 	})
 })
+
+func createStsPods(ctx context.Context, cl client.Client, sts *appsv1.StatefulSet) {
```

Comment:
Please move this to `test/utils` package, similar to the sts helper functions.

Reply:
Fixed as part of https://github.com/gardener/etcd-druid/pull/839/commits/8f6b8a607574f2dbd35ef3e2e1f7915c92c2b46d